### ISSUE TITLE
After pushing to heroku master - application error on heroku url

### ISSUE THREAD

**Issue by kikisf** [archyhq/docs#4]
After executing the following command:

`$ git push heroku master
`
I get the following error when running curl command:

```
Irinas-MacBook-Air:archy irinakozina$ curl -X POST -H "Content-Type: application/json" -d '{"payload":{ "perPage": null, "page": 1, "meta": {}, "args": {}}}' "https://mysterious-escarpment-35046.herokuapp.com"
<!DOCTYPE html>
    <html>
    <head>
      <meta name="viewport" content="width=device-width, initial-scale=1">
      <style type="text/css">
        html, body, iframe { margin: 0; padding: 0; height: 100%; }
        iframe { display: block; width: 100%; border: none; }
      </style>
    <title>Application Error</title>
    </head>
    <body>
      <iframe src="//s3.amazonaws.com/heroku_pages/error.html">
        <p>Application Error</p>
      </iframe>
    </body>
```

**Comment by kikisf** [comment]
Resolved by adding missing module 'winston-logzio'

```
Irinas-MacBook-Air:archy irinakozina$ node index.js
module.js:327
    throw err;
    ^

Error: Cannot find module 'winston-logzio'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/irinakozina/Documents/archy/node_modules/archy-sdk/lib/logger.js:4:30)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
Irinas-MacBook-Air:archy irinakozina$ npm install winston-logzio
npm WARN package.json @0.0.1 No description
npm WARN package.json @0.0.1 No repository field.
npm WARN package.json @0.0.1 No README data
npm WARN package.json @0.0.1 No license field.
npm WARN deprecated tough-cookie@2.2.2: ReDoS vulnerability parsing Set-Cookie https://nodesecurity.io/advisories/130
winston-logzio@1.0.1 node_modules/winston-logzio
├── json-stringify-safe@5.0.1
├── util@0.10.3 (inherits@2.0.1)
├── winston@1.0.2 (cycle@1.0.3, stack-trace@0.0.9, eyes@0.1.8, isstream@0.1.2, async@1.0.0, pkginfo@0.3.1, colors@1.0.3)
├── lodash@3.10.1
└── logzio-nodejs@0.3.10 (request@2.68.0)
Irinas-MacBook-Air:archy irinakozina$ node index.js
2016-09-23T01:09:38.759Z - info: App started on port: 3000
2016-09-23T01:09:50.947Z - error: unknown command name: venues
2016-09-23T01:09:50.948Z - error: available commands: events
```

**Comment by genbit** [comment]
Thanks for posting this bug. SDK is still in progress.

We just published version 0.2.0 of our sdk, it includes a fix for this issue and more.

just do

```
npm update --save archy-sdk
```
